.github/workflows/atlantis-image.yml

```diff
@@ -75,9 +75,9 @@ jobs:
           type=semver,pattern={{version}},prefix=v,enable=${{ matrix.image_type == 'alpine' }}
           type=semver,pattern={{major}}.{{minor}},prefix=v,suffix=${{ env.SUFFIX }}
           # dev
-          type=raw,value=dev,suffix=${{ env.SUFFIX }}-{{ sha }}
-          type=raw,event=push,value=dev,enable=${{ github.ref == format('refs/heads/{0}', 'main') && matrix.image_type == 'alpine' }},suffix=
           type=raw,event=push,value=dev,enable={{is_default_branch}},suffix=${{ env.SUFFIX }}
+          type=raw,event=push,value=dev,enable={{is_default_branch}},suffix=${{ env.SUFFIX }}-{{ sha }}
```

Comment:
Hi! Thanks for adding these changes.

Can you test this change in your fork to make sure this works as expected?

When the fork or is proposed in the fork to the forks main branch, the push should be disabled

When the fork pr is merged to the fork main, the build should push to the fork ghcr

Reply:
Hey, no problem. You can see the push already disabled here: https://github.com/runatlantis/atlantis/actions/runs/4932669551/jobs/8815914263?pr=3393#step:10:12

This just makes the `SUFFIX-sha` tag gated by pushes only from main. I'm not sure if I can test this properly on the fork without messing up commit history, but I will attempt.

Reply:
See: https://github.com/runatlantis/atlantis/pull/3393#issuecomment-1544426174